Kubecon na transparency report 23

cjyabraham opened the pull request:
[PR version of report](https://pr-817-cncfci.pantheonsite.io/reports/kubecon-cloudnativecon-north-america-2023/)

thetwopct reviewed (commented):
Nice work!!

A few things I noticed:

- Flags look low res - they are PNGs inside SVGs. We should have these already in the reports folder. 
- Media logos - some look low res, they are PNGs inside SVGs. We should have many of these already in the reports folder. 
- section-10 overlay disappears on desktop and I think it needs it as text is hard to read.
- Add nowrap to Media Coverage table list 1st column (on desktop) to stop media title wrapping. 

Other than that it looks great. IMO the design has far too many colours in it. Looks child like. It would look more professional with less colours. 

Addition:
- [Lighthouse](https://pagespeed.web.dev/analysis/https-pr-817-cncfci-pantheonsite-io-reports-kubecon-cloudnativecon-north-america-2023/dzfhej2s5g?form_factor=mobile) is throwing up low contrast warnings on large sections, which is not something it normally does.

cjyabraham commented:
> * Flags look low res - they are PNGs inside SVGs. We should have these already in the reports folder.

I have swapped in ones used on previous reports. It seems the USA one is still a bit fuzzy but good for now I guess.

> * Media logos - some look low res, they are PNGs inside SVGs. We should have many of these already in the reports folder.

I've swapped in ones from previous reports. Some are PNGs but are better than the ones I created.

> * section-10 overlay disappears on desktop and I think it needs it as text is hard to read.

Fixed.

> * Add nowrap to Media Coverage table list 1st column (on desktop) to stop media title wrapping.

Fixed.

> Other than that it looks great. IMO the design has far too many colours in it. Looks child like. It would look more professional with less colours.

Yes, there are a lot of different colors. Also, I don't like it when they use a pink that is almost the same as our link pink.


> Addition:
> 
> * [Lighthouse](https://pagespeed.web.dev/analysis/https-pr-817-cncfci-pantheonsite-io-reports-kubecon-cloudnativecon-north-america-2023/dzfhej2s5g?form_factor=mobile) is throwing up low contrast warnings on large sections, which is not something it normally does.

Ah, one of the pinks was failing WCAG. I've changed it slightly to pass and that seems to have resolved most of the issues although there are still two large sections getting flagged.

> * The embedded tweets are too big on desktop which then makes them impossible to read on mobile. IMO the desktop layout for them could be better and this would be solved.

Ok, will consider this for next report. Maybe we should just implement these with markup rather than a screenshot? Or use a different image for mobile than for desktop.

Honestly I could spend another day on this report to fine-tune some rough edges but I guess there's a point where it's "good enough".
